src/components/plots/styling/volcano/DimensionsRangeEditor.jsx

```diff
@@ -21,29 +21,48 @@ const VolcanoDimensionsRangeEditor = (props) => {
         onUpdate={onUpdate}
       />
       <Form.Item
-        label='Y-axis Range'
+        label='Y-Axis Range'
       >
+        <Checkbox
+          onChange={() => {
+            onUpdate({ yAxisAuto: !config.yAxisAuto });
+          }}
+          defaultChecked
+          checked={config.yAxisAuto}
+        >
+          Auto
```

Comment:
instead of using auto why dont we just keep the dimensions accurate according to the axis range when you change it? This way we won't need auto?

Reply:
I don't quite understand. Can you please explain again?